### DISCUSSION TITLE
Welcome to Ortho4XP Discussions!

### DISCUSSION THREAD

**Discussion by girotobial** [Ortho4XP/Ortho4XP#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by anthraxn8b** [comment]
Hi! I would like to become a member and work on the project. How do I get access so that I can create (feature) branches and pull requests? (I would have fork it for now, but I already have one fork of Ortho4XP - https://github.com/anthraxn8b/Ortho4XP)...
